### PR TITLE
Feat/3324 table

### PR DESCRIPTION
**Pull request by EduardoUmpierre**
### Changes
- Arrow icon added
- Table component added

### CODE REVIEW

**Comment by gateixeira** [comment]
@wagner-schwertz-poatek @jotape-poatek please review and don't need to wait for my approval.

**Review by jotape-poatek — APPROVED** [review]

**Review by wagner-schwertz-poatek — APPROVED** [review]

**Pull request merged**
